tests/conftest.py

```diff
@@ -20,5 +20,7 @@ def book_list_html():
 def book_list_html_response(book_list_html):
     body = HttpResponseBody(bytes(book_list_html, "utf-8"))
     return HttpResponse(
-        url="http://books.toscrape.com/index.html", body=body, encoding="utf-8"
+        url=ResponseUrl("http://books.toscrape.com/index.html"),
```

Comment:
why is this change needed? I think we want to support previous syntax, there is attrs convertor for that.

Reply:
Without this change the `url` attribute is a `str`.

Reply:
This shouldn't be the case, because of attrs convertor